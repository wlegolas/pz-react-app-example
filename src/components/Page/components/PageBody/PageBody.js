import React from 'react';

const PageBody = () => (
  <div className="pz-page-body">
    <h1>Page body</h1>
    <div>Application body</div>
  </div>
);

export default PageBody;
