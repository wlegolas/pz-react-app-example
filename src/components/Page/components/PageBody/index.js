export { default as PageBody } from './PageBody';
