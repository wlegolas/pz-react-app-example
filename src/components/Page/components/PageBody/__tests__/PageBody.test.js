import toJson from 'enzyme-to-json';
import React from 'react';
import PageBody from '../PageBody';

describe('<PageBody />', () => {
  describe('render()', () => {
    it('rendres the component', () => {
      const wrapper = shallow(<PageBody />);
      const componentJson = toJson(wrapper);

      expect(componentJson).to.matchSnapshot();
    });
  });
});
