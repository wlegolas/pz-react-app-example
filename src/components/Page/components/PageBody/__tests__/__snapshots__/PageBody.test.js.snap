// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageBody /> render() rendres the component 1`] = `
<div
  className="pz-page-body"
>
  <h1>
    Page body
  </h1>
  <div>
    Application body
  </div>
</div>
`;
